### ISSUE TITLE
`-e <regex>` option not working

### ISSUE THREAD

**Issue by jimhigson** [rse/stmux#28]
Simple example:

```sh
npx stmux -- [ -e i_like_pancakes "yes Error" ]
```

* **expected**: Should not be red, because nothing on the console matches our regex
* **actual**: Is red.

LMK if you don't have time to fix this, I might be able to raise a PR

**Comment by jimhigson** [comment]
Side issue - the error detection doesn't play nice with `tsc` since it prints "Found 0 errors." and it is possible but not obvious how to write a regex that matches non-zero numbers. Feature request - allow multiple regexes like: `-e Error -e "! 0 errors"`

**Comment by rse** [comment]
Yes, I'm ultra busy, so I would be glad if you could provide a PR...
